app/services/cached_commodity_service.rb

```diff
@@ -140,7 +146,7 @@ def meursing_additional_code_id
   end
 
   def cache_key
-    "_commodity-#{commodity.goods_nomenclature_sid}-#{actual_date}-#{TradeTariffBackend.currency}-#{geographical_area_id}-#{meursing_additional_code_id}"
+    "_commodity-v#{CACHE_VERSION}-#{commodity.goods_nomenclature_sid}-#{actual_date}-#{TradeTariffBackend.currency}-#{geographical_area_id}-#{meursing_additional_code_id}"
```

Comment:
Do we need this chipolata? I think we clear this nightly anyway and this would only cover us if we backed out of permutations?

Reply:
At which point we could just clear the cache manually on a deploy

Reply:
Its pretty useful to be able to just bump the cache version number when at PR invalidates it - thats pretty much how Rails default cache keys work